sensors/cyber/sensorhub-driver-sysinfo/build.gradle

```diff
@@ -0,0 +1,51 @@
+description = 'Systems Info Sensor'
+ext.details = "Driver to return Systems Information"
+version = '1.0.5'
+
+
+repositories {
+    mavenCentral()
+}
+dependencies {
+    implementation 'org.sensorhub:sensorhub-core:' + oshCoreVersion
+    implementation('org.sensorhub:sensorhub-service-swe:' + oshCoreVersion)
+//    implementation project(':sensorhub-service-video')
+//    implementation project(':sensorhub-driver-videocam')
+    testImplementation('junit:junit:4.13.1')
+    implementation("com.github.oshi:oshi-core-java11:6.4.11")
+    implementation 'com.github.pires:obd-java-api:1.0'
+    implementation 'com.sun.mail:javax.mail:1.6.2'
+    implementation 'org.apache.httpcomponents:httpclient:4.5.13'
+
+
+
+//    implementation "net.java.dev.jna:jna-jpms:${jna.version}"

```

Comment:
If not needed can we remove?  If this is here because it adds a way to help test or other reason not readily obvious add a comment around it.